### ISSUE TITLE
 Bitwarden don't copy passwords in Linux 

### ISSUE THREAD

**Issue by rl13ce** [flathub/com.bitwarden.desktop#147]

**Comment by the4anoni** [comment]
I have same issue

Debian 12 + KDE

**Comment by RaptaG** [comment]
Same issue here, on Fedora Kinoite 38 and Wayland

**Comment by almereyda** [comment]
A fix in https://github.com/bitwarden/clients/issues/6812#issuecomment-1806830091 suggests to "enable the wayland-data-control feature" of the "arboard crate".

**Comment by rderooy** [comment]
workaround to the issue:
1) view the password
2) highlight the password with the mouse
3) right click, and select "Copy"

**Comment by b1zzu** [comment]
Same here

**Comment by Ziomal12** [comment]
Had the same issue on Ubuntu 23.10 Wayland. The solution for me was to disable `socket=fallback-x11` in FlatSeal for Bitwarden. (I also have enabled `socket=wayland` but I'm not sure if it's relevant).

**Comment by eylenburg** [comment]
> Had the same issue on Ubuntu 23.10 Wayland. The solution for me was to disable `socket=fallback-x11` in FlatSeal for Bitwarden. (I also have enabled `socket=wayland` but I'm not sure if it's relevant).

This didn't work for me. (OpenSUSE Tumbleweed, KDE 5.27 Wayland)

**Comment by rderooy** [comment]
> There is a possible fix for this issue in #169
> 
> Once it lands, please try it and report whether the problem is fixed for you.

Reverting to X11 is not an actual fix. It is at best a workaround.

**Comment by ghisvail** [comment]
@rderooy Based on your comment it looks like you did not bother reading the actual commit closing the issue.

Let me quote it for you:

> Keep both Wayland and X11 socket permissions
>
> There is an issue upstream with copy and paste on pure Wayland.
>
> Users have reported that keeping the X11 socket alive solves it.
>
> **To be reverted when upstream fixes the issue for good.**

Emphasis added to be sure you caught the important line.

**Comment by eylenburg** [comment]
I don't think this should be marked as "closed" already if it doesn't work one of the two big Linux desktops. Even if the issue is possibly somewhere upstream.

**Comment by ghisvail** [comment]
> I don't think this should be marked as "closed" already if it doesn't work one of the two big Linux desktops.

This issue contains a mix of different attempt at triaging for different desktops and suggest the copy / paste feature fails to work on Linux in general.

I believe the path forward would be better served by a dedicated issue targetting KDE, with triaging and reporting solely focused on this desktop environment and with the right people involved in it.

**Comment by galloramiro** [comment]
## Problem
Copy not working

### System specs
- Operating System: Fedora Linux 39
- KDE Plasma Version: 5.27.9
- KDE Frameworks Version: 5.111.0
- Qt Version: 5.15.11
- Kernel Version: 6.6.3-200.fc39.x86_64 (64-bit)
- Graphics Platform: Wayland
- Bitwarden version: 2023.10.1 of bitwarden


### Current behavior 
1. Go to 'record'
2. Click on 'copy password'
3. Go to the place
4. ctrl + v and nothing happens
5. Go to 'record'
6. Click on show password
7. ctrl + c
8. Go to place of usage
9. ctr + v and now paste the value

### Expected behavior
1. Go to 'record'
2. Click on 'copy password'
3. Go to the place
4. ctrl + v pase the value

**Comment by ghisvail** [comment]
See #173 for a follow-up issue for KDE specifically.

**Comment by LightShaft** [comment]
Hi, I don't see any mention of it but it doesn't seem to work on Windows either. Tried to reinstall the app but I have no other idea what to try.
